packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -136,7 +136,7 @@ const Actions = {
         }
       });
       types.forEach(type => {
-        if (model.showNodesOfType[type] == null) {
+        if (model.showNodesOfType[type] === null) {
```

Comment:
```suggestion
        if (model.showNodesOfType[type] == null) {
```

Otherwise custom times don't show up in the graph

Reply:
Missed this one.